docs/learning/guidelines/typescript-coding-guidelines.md

```diff
@@ -13,6 +13,7 @@ Where possible, these guidelines are enforced through our ESLint configuration (
 6. Use `_` as a prefix for private properties.
 7. Use whole words in names when possible. Only use abbreviations where their use is common and obvious.
 8. We use "Id", "3d", "2d" rather than capital D.
+9. Capitalize i in "iModel" and "iTwin" to match PascalCase or camelCase. Always use capital M and T.
```

Comment:
Be more specific: PascalCase for types, camelCase for variables / function parameters.

Reply:
I changed it to be a bit clearer while trying to avoid being long or redundant.